extras/nb_table_exporter.py

```diff
@@ -119,5 +99,111 @@ def get_instruction_id(t_id, connection):
         export_instructions(instruction_id, connection)
 
 
+def export_all_task_records():
+    conn = get_conn("neurobooth", False)
+    path = os.path.join(write_path, 'tasks')
+
+    task_ids = get_task_ids_for_collection("test_mvp_030", conn)
+    for t_id in task_ids:
+        table_task = Table("nb_task", conn=conn)
+        task_df = table_task.query(where=f"task_id = '{t_id}'")
+        (feature_of_interest,) = task_df["feature_of_interest"]
+        (device_ids,) = task_df["device_id_array"]
+        (sensor_ids,) = task_df["sensor_id_array"]
+        (stimulus_id,) = task_df["stimulus_id"]
+        (instr_id,) = task_df["instruction_id"]
+
+        task_dict = {}
+        task_dict['task_id'] = t_id
+        task_dict['feature_of_interest'] = feature_of_interest
+        task_dict['stimulus_id'] = stimulus_id
+        task_dict['instruction_id'] = instr_id
+        task_dict['device_id_array'] = device_ids
+        task_dict["sensor_id_array"] = sensor_ids
+        task_dict['arg_parser'] = 'iout.stim_param_reader.py::RawTaskParams'
+
+        filename = os.path.join(path, t_id + ".yml")
+        with open(filename, 'w') as f:
+            yaml.dump(task_dict, f, sort_keys=False)
+
+
+def export_all_device_records():
+    conn = get_conn("neurobooth", False)
+    path = os.path.join(write_path, 'devices')
+
+    task_ids = get_task_ids_for_collection("test_mvp_030", conn)
+    for t_id in task_ids:
+        table_task = Table("nb_task", conn=conn)
+        task_df = table_task.query(where=f"task_id = '{t_id}'")
+        (device_ids,) = task_df["device_id_array"]
+
+        table_device = Table("nb_device", conn=conn)
+        for device_id in device_ids:
+            device_df = table_device.query(where=f"device_id = '{device_id}'")
+            (device_sn,) = device_df["device_sn"]
+            (device_name,) = device_df["device_name"]
+            (device_location,) = device_df["device_location"]
+            (wearable,) = device_df["wearable_bool"]
+            (device_make,) = device_df["device_make"]
+            (device_model,) = device_df["device_model"]
+            (device_firmware,) = device_df["device_firmware"]
+            (sensor_id_array,) = device_df["sensor_id_array"]
+
+            dev_dict = {}
+            dev_dict['device_id'] = device_id
+            dev_dict['device_sn'] = device_sn
+            dev_dict['device_name'] = device_name
+            dev_dict['device_location'] = device_location
+            dev_dict['wearable_bool'] = wearable
+            dev_dict["device_make"] = device_make
+            dev_dict["device_model"] = device_model
+            dev_dict["device_firmware"] = device_firmware
+            dev_dict["sensor_ids"] = sensor_id_array
+            dev_dict['arg_parser'] = 'iout.stim_param_reader.py::DeviceArgs'
+
+            filename = os.path.join(path, device_id + ".yml")
```

Comment:
Very minor point, but we could have a `write_yaml(Dict) -> None` function since this shows up a few times.

Reply:
I will remove this module when this work is done since all these tables will be gone